pyxem/signals/polar_diffraction2d.py

```diff
@@ -130,6 +130,51 @@ def get_angular_power(self, mask=None, normalize=True, inplace=False, **kwargs):
         fourier_axis.scale = 1
         return power
 
+    def get_pearson_correlation(self, mask = None, selectk=False, kmin=0, kmax=0, inplace=False, **kwargs):
+        """Returns the pearson rotational correlation in the form of a Signal2D class.
+
+        Parameters
+        ----------
+        mask: Numpy array
+            A bool mask of values to ignore of shape equal to the signal shape.
+        selectk: bool
+            Select k range for correlation over a ring segment
+        kmin: float
+            minimum k value in corresponding unit for segment correlation
+        kmax: float
+            maximum k value in corresponding unit for segment correlation
```

Comment:
I don't think you should have `selectk` as a kwargs like this. Instead you can use` kmin=None` and `kmax=None` as default values then run a pair of blocks to do `if kmin is not None`

Reply:
Good idea! I'll implement the change.